blog/2024-01-08-cloudstack-and-gitops-at-scale/index.md

```diff
@@ -119,7 +119,4 @@ Our expectation is essentially all workloads in the enterprise
 can/could be managed with CloudStack. Web applications, Kubernetes
 clusters, databases, etc.
 
-<br/>
-<br/>
-
-## More about AT&T: https://www.att.com/
+<h4>More about AT&T: <a href="https://www.att.com">https://www.att.com</a></h4>
```

Comment:
why is this needed? extra html in the md files seems to me to be avoided as much as possible.

Reply:
It shows on the right-side table of contents otherwise.